plugins/source/postgresql/client/spec.go

```diff
@@ -0,0 +1,14 @@
+package client
+
+type Spec struct {
+	ConnectionString string   `json:"connection_string,omitempty"`
+	PgxLogLevel      LogLevel `json:"pgx_log_level,omitempty"`
+	CDC              bool     `json:"cdc,omitempty"`
```

Comment:
I think users should be able to specify the tables they want to replicate here :)

Reply:
They can specify it via the top level source `tables`. it is working due to the dynamic table features.

Reply:
ah of course, yeah 👍